CMakeLists.txt

```diff
@@ -328,6 +329,24 @@ if(ENABLE_LUA)
 	add_definitions(-DSCRIPTING_ENABLED=1)
 endif()
 
+if(USING_CONAN AND (MINGW AND CMAKE_HOST_UNIX))
+	# Hack for workaround https://github.com/conan-io/conan-center-index/issues/15405
+	# Remove once it will be fixed
+	execute_process(COMMAND
+		bash -c "grep -rl Mf ${CONAN_INSTALL_FOLDER} | xargs sed -i 's/Mf/mf/g'"
+	)
+	# Hack for workaround ffmpeg broken linking (conan ffmpeg forgots to link to ws2_32)
+	# Remove once it will be fixed
+	execute_process(COMMAND
+		bash -c "grep -rl secur32 ${CONAN_INSTALL_FOLDER} | xargs sed -i 's/secur32/secur32 ws2_32/g'"
```

Comment:
I guess this will be executed each time you run cmake configuration? then it will "ruin" Conan config as on second run you'll receive `secur32 ws2_32 ws2_32`. Usually specifying the same library for linking multiple times is not an issue, hope it won't break anything.

Reply:
I tried it 3 times, cmake is smart enough to deduplicate libraries. But I can change sed line to 2 sed lines, which will be run only once.

Reply:
Replace sed line with 2 lines without side effect